src/mazzaroth.js

```diff
@@ -90,40 +95,23 @@ Examples:
 `
 clientCommand('transaction-call', transactionCallDesc, transactionOptions.concat(callOptions),
   (val, options, client) => {
+    const blockExpiration = defaultExpiration | options.transaction_expire_after
     const action = {
       channelID: options.channel_id || defaultChannel,
       nonce: (options.nonce || Math.floor(Math.random() * Math.floor(1000000000))).toString(),
       category: {
         enum: 1,
         value: {
           function: val,
-          parameters: callArgs
+          arguments: callArgs
```

Comment:
I thought this was parameters ? maybe I am thinking of the wrong thing

Reply:
No the actual values being passed in the call action are now called "arguments":
https://github.com/kochavalabs/mazzaroth-xdr/blob/develop/idl/transaction.x#L14

You may be thinking of the ABI which refers to the "Parameter" in the function signature:
https://github.com/kochavalabs/mazzaroth-xdr/blob/develop/idl/abi.x#L15